.github/workflows/deploy.yml

```diff
@@ -47,27 +47,30 @@ on: # yamllint disable-line rule:truthy
         type: string
         description: 'Input db password'
         required: true
-        default: 'mysecretpassword'
+        default: 'dppass'
```

Comment:
it's ok for now, but these vars should be specified as secrets not inputs and described in documentation

Reply:
slow loading during running playbook in case of

Reply:
ansible works fine with more than 1000 vars